eng/containers/UbuntuNetCoreKeyring/Dockerfile

```
@@ -8,6 +8,7 @@ ENV \
     # Do not generate certificate
     DOTNET_GENERATE_ASPNET_CERTIFICATE=false \
     # SDK version
+    DOTNET_SDK_VERSION_8_0=8.0.405\
     DOTNET_SDK_VERSION_6_0=6.0.413 \
     DOTNET_SDK_VERSION_3_1=3.1.416 \
```

```suggestion
    DOTNET_SDK_VERSION_6_0=6.0.413 \
    DOTNET_SDK_VERSION_3_1=3.1.416 \
```

We should remove any references to these, as they're no longer supported.

---

We still run test suites on net60, so my thought was we should keep that until we don't. net31 makes sense to remove though.

---

We are in the process of migrating the test targets over to net8+net9.    We can cover this there if needed.

//cc: @m-redding

---

Yes I can remove this as part of that process!

---

Sounds good - let's update it in one shot when we do that then. This image is already published to our ACR container anyway. 

@m-redding Feel free to reach out when you get to this if you need anything from me!